create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusConverter.java

```diff
@@ -83,11 +84,34 @@ private EntityDescription generateEntityDescription() {
         EntityDescription entityDescription = new EntityDescription();
         entityDescription.setReference(generateReference());
         entityDescription.setMainTitle(extractMainTitle());
+        entityDescription.setAbstract(extractMainAbstract());
         entityDescription.setContributors(generateContributors());
         entityDescription.setTags(generatePlainTextTags());
         return entityDescription;
     }
 
+    private String extractMainAbstract() {
+        return getMainAbstract().map(this::marshallAbstract).orElse(null);
+    }
+
+    private Optional<AbstractTp> getMainAbstract() {
+        return getAbstracts().stream().filter(this::isOriginalAbstract).findFirst();
+    }
+
+    private List<AbstractTp> getAbstracts() {
+        return docTp.getItem().getItem().getBibrecord().getHead().getAbstracts().getAbstract();
+    }
+
+    private boolean isOriginalAbstract(AbstractTp abstractTp) {
+        return abstractTp.getOriginal() == YesnoAtt.Y;
```

Comment:
YesnoAtt.Y that's a wild one ;) 
Kanskje man burde snu statement.
`YesnoAtt.Y == abstractTp.getOriginal()`

Reply:
fikset